sass/_authorinfo.scss

```diff
@@ -0,0 +1,54 @@
+.author__info {
+  margin: 30px 10% 20px;
+  text-align: center;
+  > h1 {
+    font-size: 1.5em;
+    color: #106DB3;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+}
+.author__pic {
+  background-image: url('./images/dandysir.jpg');
+  background-size: cover;
+  background-position: 50% 50%;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100px;
+  height: 100px;
+  border-radius: 100%;
+}
+.author__picOverlay {
+  background:rgba(23, 107, 175, .5);
+  text-align: center;
+  opacity: 1;
+  width:100%;
+  min-height:100%;
+  border-radius: 100%;
+  &:hover {
+    opacity: 0;
+  }
+}
+.author__links {
+  list-style: none;
+  // margin-top: 0;
+  padding-left: 0;
+  > li {
+    display: inline-block;
+    padding: 0 10px 10px;
+    font-size: 1.1em;
+    color: #CCCCCC;
```

Comment:
:+1:

On Wed, Jul 8, 2015 at 5:01 PM, Jamie Simper <notifications@github.com>
wrote:

> In sass/_authorinfo.scss
> <https://github.com/jasimper/3.4-Surf_and_Paddle/pull/1#discussion_r34198084>
> :
>
> > +  width:100%;
> > +  min-height:100%;
> > +  border-radius: 100%;
> > +  &:hover {
> > +    opacity: 0;
> > +  }
> > +}
> > +.author__links {
> > +  list-style: none;
> > +  // margin-top: 0;
> > +  padding-left: 0;
> > +  > li {
> > +    display: inline-block;
> > +    padding: 0 10px 10px;
> > +    font-size: 1.1em;
> > +    color: #CCCCCC;
>
> I'm anti-shorthand for hex values. I'm not sure why, but I like all of
> them being 6 characters.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jasimper/3.4-Surf_and_Paddle/pull/1/files#r34198084>.
>



-- 
Robert Pearce
http://www.robertwpearce.com
https://www.linkedin.com/in/robertwpearce
m-US: 1.843.568.9080
m-UK: +44 7477 424374